app/uk/gov/hmrc/customs/rosmfrontend/domain/CdsLoggedInUser.scala

```diff
@@ -50,5 +50,6 @@ case class LoggedInUserWithEnrolments(
   internalId: Option[String],
   enrolments: Enrolments,
   email: Option[String],
-  groupId: Option[String]
+  groupId: Option[String],
+  credId: Option[String]
```

Comment:
Is this the CredId from GG Auth (i.e. first field on the Auth Wizard Stub)?    I'm surprised we don't have that already.

Reply:
Yes, it's this CredId. I'm surprised too.